app/registries/foreman/plugin.rb

```diff
@@ -157,6 +157,14 @@ def initialize(id)
       @status_extension = nil
     end
 
+    def engine
+      @engine ||= Rails::Engine.find(path) if path
+    end
+
+    def migrations_paths
+      engine.paths['db/migrate'].existent
```

Comment:
The test failures seem to be related to the fact that `engine` can be `nil`, or at least in tests

Reply:
Yeah, then skipping the check is a valid option, as without engine, the plugin has no migrations. Done that, by early return.